spec/requests/linking/link_defendant_maat_reference_spec.rb

```diff
@@ -6,11 +6,16 @@
   let(:nino) { 'JC123456A' }
   let(:defendant_id) { '41fcb1cd-516e-438e-887a-5987d92ef90f' }
   let(:maat_reference) { '2123456' }
+  let(:defendant_identifier) { nino }
   let(:params) do
     {
-      id: nino,
-      defendant_id: defendant_id,
-      maat_reference: maat_reference
+      link_attempt:
+      {
+        id: nino,
+        defendant_id: defendant_id,
+        maat_reference: maat_reference,
+        defendant_identifier: defendant_identifier
+      }
```

Comment:
if the defendant :id is the nino (or asn), why can we not just use that in the controllers generally. 

i'd rename the defendant_identifier, as previously described as this can help show up these oddities and/or duplications

Reply:
👍 I have removed defendant identifier and added defendant_asn_or_nino instead of id.